solidity/contracts/helpers/TestStandardPoolConverter.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+import "../converter/types/standard-pool/StandardPoolConverter.sol";
+
+contract TestStandardPoolConverter is StandardPoolConverter {
+    uint256 public currentTime = 1;
```

Comment:
We would need to refactor it with the `TestTime` change once it's merged

Reply:
True.

Reply:
Done